### PR TITLE
[BrokenLinksH2]

### PR DESCRIPTION
**Pull request by DhurataJ**
Fixing broken links, outdated links related to AsptNetDocs repository

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=611) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=611) before we can accept your contribution.<br/><br/>:x: DhurataJ [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=611)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=611) it.</sub>

**Comment by DhurataJ** [comment]
**Global effort to fix broken links**

@Rick-Anderson 

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ The [finished application](https://github.com/dotnet/AspNetDocs/tree/main/aspnet
...
 The development machine must have the following software:
 
-- [Windows 10 Anniversary Update](https://www.microsoft.com/software-download/windows10/) (or higher) or [Windows Server 2016](https://www.microsoft.com/cloud-platform/windows-server) (or higher)
+- [Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10) (or higher) or [Windows Server 2016](https://www.microsoft.com/cloud-platform/windows-server) (or higher)
 - [Docker for Windows](https://docs.docker.com/docker-for-windows/) - version Stable 1.13.0 or 1.12 Beta 26 (or newer versions)
 - [Visual Studio 2017](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017)
 
 > [!IMPORTANT]
-> If you are using Windows Server 2016, follow the instructions for [Container Host Deployment - Windows Server](https://msdn.microsoft.com/virtualization/windowscontainers/deployment/deployment).
+> If you are using Windows Server 2016, follow the instructions for [Container Host Deployment - Windows Server](https://docs.microsoft.com/en-us/virtualization/windowscontainers/).
```
Comment (Rick-Anderson) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md:39]:
```suggestion
> If you are using Windows Server 2016, follow the instructions for [Container Host Deployment - Windows Server](https://docs.microsoft.com/virtualization/windowscontainers/).
```

File: aspnet/mvc/overview/getting-started/introduction/adding-search.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ The following code is a LINQ query that retrieves all the genres from the databa
...
 
-The code uses the `AddRange` method of the generic `List` collection to add all the distinct genres to the list. (Without the `Distinct` modifier, duplicate genres would be added — for example, comedy would be added twice in our sample). The code then stores the list of genres in the `ViewBag.MovieGenre` object. Storing category data (such a movie genres) as a [SelectList](https://msdn.microsoft.cus/library/system.web.mvc.selectlist(v=vs.108).aspx) object in a `ViewBag`, then accessing the category data in a dropdown list box is a typical approach for MVC applications.
+The code uses the `AddRange` method of the generic `List` collection to add all the distinct genres to the list. (Without the `Distinct` modifier, duplicate genres would be added — for example, comedy would be added twice in our sample). The code then stores the list of genres in the `ViewBag.MovieGenre` object. Storing category data (such a movie genres) as a [SelectList](https://docs.microsoft.com/en-us/dotnet/api/system.web.mvc.selectlist) object in a `ViewBag`, then accessing the category data in a dropdown list box is a typical approach for MVC applications.
```
Comment (Rick-Anderson) [inline aspnet/mvc/overview/getting-started/introduction/adding-search.md:125]: remove all language , that is, /en-us from URLs

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
remove en-us/

**Review by Rick-Anderson — APPROVED** [review]

File: aspnet/visual-studio/overview/2013/aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes.md [hunk 3]
```diff
@@ -257,7 +256,7 @@ Entity Framework has been updated to version 6.1 for both runtime and tooling. E
 <a id="owin"></a>
 ### Microsoft OWIN components
 
-There were lots of bugs that were fixed in this release. Please see the [release notes for the 2.1.0 release](https://katanaproject.codeplex.com/releases/view/113281) for more detailed information.
+There were lots of bugs that were fixed in this release.
```
Comment (TheMythoclast) [inline aspnet/visual-studio/overview/2013/aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes.md:259]: I will attempt to find some patch notes. There were a lot of bugs that were fixed in this release.

**Pull request merged**